packages/cerebral-provider-firebase/src/helpers.js

```diff
@@ -83,3 +83,30 @@ export function createUser (user) {
     photoURL: user.photoURL
   }
 }
+
+export function createReturnPromise (returnPromise, path) {
```

Comment:
Hm... this is done several places so is a lot of refactor and documentation needs to be written. Do not consider it blocking, but will make note of it here, so we can do this in later PR... a good thing to improve even after release